### PR TITLE
[New] Add angular-css-injector w/ git auto-update

### PR DESCRIPTION
**Pull request by Billy4195**
Pull request for issue: #7598
Related issue(s): 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/GabrielDelepine/angular-css-injector
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 5
   - Count of stars: 163
   - Count of forks: 85
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by Billy4195** [comment]
@sashberd Done. If I miss some fields, please tell me.

File: ajax/libs/angular-css-injector/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ],
+  "author": "Gabriel Delépine",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GabrielDelepine/angular-css-injector"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/GabrielDelepine/angular-css-injector.git",
```
Comment (sufuf3) [inline ajax/libs/angular-css-injector/package.json]: Please use `git` protocol to let bot to get the files.
Thanks.

**Comment by sufuf3** [comment]
@sashberd Would you tell us that why we need to add the [lib](https://www.npmjs.com/package/angular-css-injector) from npm which publish from other people but not author?
I think we always add the lib from author's lib, don't we?

Thanks.

File: ajax/libs/angular-css-injector/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "name": "angular-css-injector",
+  "description": "angularJS service to load dynamically CSS files",
+  "version": "1.4.0",
+  "filename": "angular-css-injector.min.js",
+  "keywords": [
+    "angular",
+    "css",
+    "injector"
+  ],
+  "author": "Gabriel Delépine",
```
Comment (sufuf3) [inline ajax/libs/angular-css-injector/package.json]: Let's modify to `"Gabriel Delépine <yappli@github.com>"` and let it as the same with upstream.
https://github.com/GabrielDelepine/angular-css-injector/blob/main/bower.json#L7
Thanks

File: ajax/libs/angular-css-injector/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "angular-css-injector",
+  "description": "angularJS service to load dynamically CSS files",
+  "version": "1.4.0",
+  "filename": "angular-css-injector.min.js",
+  "keywords": [
+    "angular",
+    "css",
```
Comment (sufuf3) [inline ajax/libs/angular-css-injector/package.json]: Let's add more keyword https://github.com/GabrielDelepine/angular-css-injector/blob/main/bower.json#L11
Thanks

**Comment by Billy4195** [comment]
@sufuf3 I changed the fields in package.json. Thanks.

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer Please help review this PR, thanks.

File: ajax/libs/angular-css-injector/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-css-injector",
+  "filename": "angular-css-injector.min.js",
+  "description": "angularJS service to load dynamically CSS files",
+  "version": "1.4.0",
+  "keywords": [
+    "angular",
+    "css",
+    "injector",
+    "angularjs"
```
Comment (extend1994) [inline ajax/libs/angular-css-injector/package.json]: Let's also add `angularjs-service` that also shows on git repo.

File: ajax/libs/angular-css-injector/package.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "author": "Gabriel Delépine  <yappli@github.com>",
```
Comment (extend1994) [inline ajax/libs/angular-css-injector/package.json]: Please remove extra space between name and e-mail.

**Comment by Billy4195** [comment]
@extend1994 Updated! 😄

**Pull request merged**